pkg/rancher-desktop/main/__tests__/deploymentProfiles.spec.ts

```diff
@@ -0,0 +1,326 @@
+/* eslint object-curly-newline: ["error", {"consistent": true}] */
+
+import fs from 'fs';
+import os from 'os';
+import path from 'path';
+
+import * as settings from '@pkg/config/settings';
+import { readDeploymentProfiles } from '@pkg/main/deploymentProfiles';
+import { spawnFile } from '@pkg/utils/childProcess';
+import Logging from '@pkg/utils/logging';
+import { RecursivePartial } from '@pkg/utils/typeUtils';
+
+// for some reason `import nativeReg...` => undefined when run in jest on Windows
+// import nativeReg from 'native-reg';
+const nativeReg = require('native-reg');
+
+const console = Logging.deploymentProfile;
+
+// Note that we can't modify the HKLM hive without admin privileges,
+// so this whole test will just work with the user's HKCU hive.
+const REG_PATH_START = ['SOFTWARE', 'Rancher Desktop'];
+const FULL_REG_PATH_START = ['HKEY_CURRENT_USER'].concat(REG_PATH_START);
+const REGISTRY_PATH_PROFILE = REG_PATH_START.concat('TestProfile');
+
+const NON_PROFILE_PATH = FULL_REG_PATH_START.join('\\');
+const FULL_PROFILE_PATH = FULL_REG_PATH_START.concat('TestProfile').join('\\');
+
+const describeWindows = process.platform === 'win32' ? describe : describe.skip;
+
+let testDir = '';
+let regFilePath = '';
+
+async function clearRegistry() {
+  try {
+    await spawnFile('reg', ['DELETE', `HKCU\\${ REGISTRY_PATH_PROFILE.join('\\') }`, '/f']);
+  } catch {
+    // Ignore any errors
+  }
+}
+
+async function installInRegistry(regFileContents: string) {
+  const BOM = ''; // \uFEFF';
```

Comment:
Okay, since we're using `latin1`, then there should _not_ be a BOM
```suggestion
```

Reply:
Yes